### DISCUSSION TITLE
Android home/overview buttons effect after click on ads

### DISCUSSION THREAD

**Discussion by orah29** [Poing-Studios/godot-admob-editor#101] [Q&A]
After clicking on ads,
- if I press on overview or home buttons, the app closes;
- if I press on back button, I return to ads. Then I close the ads to return to the app.
Is it a normal behavior of the plugin?
Thanks in advance

**Comment by gumaciel** [comment]
See https://github.com/Poing-Studios/godot-admob-android/issues/187
